config-model/src/main/java/com/yahoo/vespa/model/content/IndexedHierarchicDistributionValidator.java

```diff
@@ -97,10 +97,10 @@ static public void validateThatReadyCopiesIsCompatibleWithRedundancy(String clus
         }
         if (totalReadyCopies % groupCount != 0) {
             throw new IllegalArgumentException(getErrorMsgPrefix(clusterName) + "Expected equal amount of ready copies per group, but " +
-                                totalReadyCopies + " ready copies is specified with " + groupCount + " groups");
+                                               totalReadyCopies + " ready copies is specified with " + groupCount + " groups");
         }
         if (totalReadyCopies == 0) {
-            System.err.println(getErrorMsgPrefix(clusterName) + "Warning. No ready copies configured. At least one is recommended.");
+            throw new IllegalArgumentException(getErrorMsgPrefix(clusterName) + "Warning. No ready copies configured. At least one is recommended.");
```

Comment:
Perhaps we should re-phrase to "At least one is required"?

Reply:
done